### ISSUE TITLE
Replace Entrance animations with Animate On Scroll Library (AOS) 

### ISSUE THREAD

**Issue by vovkasolovev** [elementor/elementor#9855]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
Similar issue https://github.com/elementor/elementor/issues/5781

- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request is going to solve? Please describe.**
The existing animation set of Entrance animations has too cheap and unprofessional look. The set of animations offered is scarce. The movements in the animation are too big.The animations will look more interesting and professional when users will have a tool to make animations not so expressed as they have now. For example, a shift for 5-30 pixels looks better than movement on 100% of the width of the container.

**Describe the solution you'd like**
I suggest to replace Entrance animations (which apparently are based on library animate.css) with the AOS library. Animate On Scroll has more features and has a suitable way for Elementor to describe animations with key=value attributes.

https://github.com/michalsnik/aos/tree/v2

The AOS have settings for everything: offset, delay, duration, easing, anchor, triggers and repeating for every item or as global settings. Using AOS can give users a tool to customize the Entrance animations with an interface similar to what is now Elementor have in the Scroll Effects. Together, appearance Animations and Scrolling effects can greatly increase the possibilities of animations when creating attractive pages.

**Additional context**
For backward compatibility, when implementing AOS, you can completely replace existing animations with similar ones from the new system, so that nothing will change for users who use the existing system.

AOS Author demo: https://michalsnik.github.io/aos/
Author article about AOS: https://css-tricks.com/aos-css-driven-scroll-animation-library/

**Comment by alem-87** [comment]
Very necessary!

**Comment by Sierrv15** [comment]
I completely agree! Change in entrance animations within Elementor is very necessary! Any news about it?

**Comment by jmora2ec** [comment]
agree entrance animation looks old school.

**Comment by be2clack** [comment]
I completely agree!!!!!!!!!

**Comment by ofmarconi** [comment]
+1

**Comment by SolunaCG** [comment]
Any update here? It´s crazy that this was already requested in 2019. I know there are some other threads about this topic and I´ve been looking for ways to improve these entrance animations myself but I haven´t found out a good solution so far... 

AOS looks amazing and it´d finally help us to make the Elementor websites look more modern in terms of animations! Especially because they seem to be triggered everytime the element enters the viewport and not just once like the Elementor entrance animations.

**Comment by bryllelim** [comment]
It does look really cheap. The movements are too much. 
Any plans on updating this?
